code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1078,8 +1078,12 @@
 	if((H.status_flags & GODMODE))
 		return
 
-	if(!breath || (breath.total_moles() == 0))
-		if(H.reagents.has_reagent("epinephrine"))
+	var/lungs = H.getorganslot("lungs")
+	if(!lungs)
```

Comment:
A species that doesn't breathe shouldn't have the lungs in a first place, but i'm too lazy to make different organ sets for each subspecies possible. At least for now.